Add counter.set_value service (#27345)
home-assistant/home-assistant.github.io: source/_integrations/counter.markdown

| ---------------------- | -------- | ----------- |
| `entity_id`            |      no  | Name of the entity to take action, e.g., `counter.my_custom_counter`. |

### Service `counter.set_value`

> With this service the value of the counter to a specific value.

Reviewer: ``` suggestion
With this service, set the value of the counter to a specific value.
```

Author: Hmm, don't like my original :D but not too sure about that suggestion either.

How about:

```suggestion
This service allows setting the counter to a specific value.
```

> With this service the value of the counter to a specific value.

Reviewer: ``` suggestion
With this service, set the counter's value to a specific value.
```
